src/libs/antares/study/parameters.h

```diff
@@ -505,6 +505,13 @@ class Parameters final
     SimplexOptimization simplexOptimizationRange;
     //@}
 
+    //! Transmission capacities options for links between area type 1 and area type 2, for adequacy
```

Comment:
maybe detail a bit the doc.
like no ntc for link between physical ares inside and outside adq patch
no ntc for link between physical areas outside adq patch